eventuate-messaging-partition-management-tests/build.gradle

```diff
@@ -1,6 +1,8 @@
 apply plugin: PublicModulePlugin
 
 dependencies {
+    compile 'com.google.collections:google-collections:1.0'
```

Comment:
Why is this dependency needed? There are not other changes in this module?

Reply:
eventuate-common-coordination-leadership does not contain 'org.apache.curator:curator-recipes' dependency anymore. It is moved to  eventuate-common-coordination-leadership-zookeeper. But there were used immutable collections which curator-recipes provided indirectly.

Reply:
Not sure that this dependency is really needed. e.g. `Arrays.asList`

Reply:
ok, I will fix